packages/react-aria-components/src/Select.tsx

```diff
@@ -62,7 +62,13 @@ export interface SelectRenderProps {
   isRequired: boolean
 }
 
-export interface SelectProps<T extends object = {}> extends Omit<AriaSelectProps<T>, 'children' | 'label' | 'description' | 'errorMessage' | 'validationState' | 'validationBehavior' | 'items'>, RACValidation, RenderProps<SelectRenderProps>, SlotProps {}
+export interface SelectProps<T extends object = {}> extends Omit<AriaSelectProps<T>, 'children' | 'label' | 'description' | 'errorMessage' | 'validationState' | 'validationBehavior' | 'items'>, RACValidation, RenderProps<SelectRenderProps>, SlotProps {
+  /**
+   * Temporary text that occupies the select when it is empty.
+   * @default 'Select an item' in English
```

Comment:
Any particular need to mention "in English" here? Is it to point out that localization will happen automatically aka "Select an item" is only when `en-US`/english locales? I think it should be fine if omitted but not really bothered either way

Reply:
Maybe "Select an item" (localized) ?